com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -366,53 +348,112 @@ internal Vector3 Scale
 
-        private readonly NetworkVariable<NetworkTransformState> m_ReplicatedNetworkState = new NetworkVariable<NetworkTransformState>(new NetworkTransformState());
+        /// <summary>
+        /// Used to know when we are resetting the dirty flag (i.e. we sent the state)
+        /// </summary>
+        internal class NetworkTransformVariable : NetworkVariable<NetworkTransformState>
+        {
+            public override void ResetDirty()
+            {
+                base.ResetDirty();
 
-        private NetworkTransformState m_LocalAuthoritativeNetworkState;
+                // Now clear our bitset and prepare for next network tick state update
+                Value.ClearBitSetForNextTick();
+            }
+
+            public NetworkTransformVariable(NetworkTransformState value = default,
+                NetworkVariableReadPermission readPerm = DefaultReadPerm,
+                NetworkVariableWritePermission writePerm = DefaultWritePerm)
+                : base(value, readPerm, writePerm)
+            {
+            }
+        }
+
+        /// <summary>
+        /// We have two internal NetworkVariables.
+        /// One for server authoritative and one for "client/owner" authoritative.
+        /// </summary>
+        private readonly NetworkTransformVariable m_ReplicatedNetworkStateServer = new NetworkTransformVariable(new NetworkTransformState(), NetworkVariableReadPermission.Everyone, NetworkVariableWritePermission.Server);
+        private readonly NetworkTransformVariable m_ReplicatedNetworkStateOwner = new NetworkTransformVariable(new NetworkTransformState(), NetworkVariableReadPermission.Everyone, NetworkVariableWritePermission.Owner);
 
-        private const int k_DebugDrawLineTime = 10;
+
+        internal NetworkVariable<NetworkTransformState> ReplicatedNetworkState
+        {
+            get
+            {
+                if (!IsServerAuthoritative())
+                {
+                    return m_ReplicatedNetworkStateOwner;
+                }
+
+                return m_ReplicatedNetworkStateServer;
+            }
+        }
+
+        private NetworkTransformState m_LocalAuthoritativeNetworkState;
 
         private bool m_HasSentLastValue = false; // used to send one last value, so clients can make the difference between lost replication data (clients extrapolate) and no more data to send.
 
 
-        private BufferedLinearInterpolator<float> m_PositionXInterpolator; // = new BufferedLinearInterpolatorFloat();
-        private BufferedLinearInterpolator<float> m_PositionYInterpolator; // = new BufferedLinearInterpolatorFloat();
-        private BufferedLinearInterpolator<float> m_PositionZInterpolator; // = new BufferedLinearInterpolatorFloat();
-        private BufferedLinearInterpolator<Quaternion> m_RotationInterpolator; // = new BufferedLinearInterpolatorQuaternion(); // rotation is a single Quaternion since each euler axis will affect the quaternion's final value
```

Comment:
I'd still keep the comment to explain why this one is different from the others

Reply:
My bad... it got deleted be accident.  Good catch!